test/integration/targets/bigip_gtm_wide_ip/tasks/environment/issue-persistence.yaml

```diff
@@ -0,0 +1,124 @@
+---
+
+- name: Issue Persistence - Include issue variables
+  include_vars:
+    file: issue-persistence.yaml
+
+- name: Issue Persistence - Provision GTM
+  bigip_provision:
+    module: gtm
+  tags:
+    - module-provisioning
+
+- name: Issue Persistence - Collect BIG-IP facts
+  bigip_device_facts:
+    include: system-info
+  register: f
```

Comment:
move it to main.yml and apply the version check to the entire task import, also gtm support for v11 is deprecated so it is not really needed

Reply:
removed.